frontend/Node.js

```diff
@@ -10,6 +10,8 @@
  */
 'use strict';
 
+const PropTypes = require('prop-types');
```

Comment:
const/var is inconsistent but I don't care

Reply:
Yeah. This is the codemod's doing. It tries to match what's in the file (based on the presence of other `const`/`let` keywords). We're inconsistent in a lot of places in the DevTools codebase, unfortunately.